internal/driver/commands.go

```diff
@@ -215,7 +218,7 @@ var pprofVariables = variables{
 		"For memory profiles, report average memory per allocation.",
 		"For time-based profiles, report average time per event.")},
 	"sample_index": &variable{stringKind, "", "", helpText(
-		"Sample value to report (0-based index or name)",
+		"Sample value to report (<x> is 0-based index or name)",
```

Comment:
This doesn't seem related to this change.

Reply:
Reverted.